pydarn/utils/geo.py

```diff
@@ -29,178 +27,17 @@
 # and conditions of version 3 of the GNU General Public License,
 # supplemented by the additional permissions listed below.
 #
-#
-
-
 """
-This module is used for handling coordinates of a specified radar
-in AACGMv2 or geographic coordinates
+This module focuses on geographic calculations used for converting gate
+information to geographic location
 """
-
-import datetime as dt
 import numpy as np
```

Comment:
Sorry for single comment, but installing this branch to test with the website: looks like an import line is missing between here are the develop branch?

`from pydarn import VHModels, EARTH_EQUATORIAL_RADIUS`